src/igvfd/tests/test_upgrade_cell_line.py

```diff
@@ -20,3 +20,18 @@ def test_cell_line_upgrade_3_4(upgrader, cell_line_v3):
     assert 'alternate_accessions' not in value
     assert 'treatments' not in value
     assert value['schema_version'] == '4'
+
+
+def test_cell_line_upgrade_4_5(upgrader, cell_line_v4, cell_line_v4_unknown, cell_line_v4_90_or_above):
+    value = upgrader.upgrade('cell_line', cell_line_v4, current_version='4', target_version='5')
+    assert value['lower_bound_age'] == 10 and value['upper_bound_age'] == 10
+    assert 'life_stage' not in value
+    assert value['schema_version'] == '5'
+    value = upgrader.upgrade('cell_line', cell_line_v4_unknown, current_version='4', target_version='5')
+    assert 'life_stage' not in value
+    assert 'age' not in value
+    assert value['schema_version'] == '5'
+    value = upgrader.upgrade('cell_line', cell_line_v4_90_or_above, current_version='4', target_version='5')
+    assert 'life_stage' not in value
+    assert value['lower_bound_age'] == 90 and value['upper_bound_age'] == 90
```

Comment:
I asked @gabdank this as well.  i am confused about how we are going to handle 90 and above now.  Is it 90 to 120?

Reply:
NVM, I have re-read the instructions again and I understand.  I am not sure we have a clean way of enforcing this plan.